node/tests/commands/deploy.spec.js

```diff
@@ -2,13 +2,21 @@ const deploy = require('../../src/commands/deploy');
 const DeployUtils = require('../../src/lib/deploy-utils');
 const { options } = require('../../src/config/constants');
 
-const { REQUIRES_APPROVAL } = options;
+const { ENVIRONMENT_NAME, PROJECT_ID, ORGANIZATION_ID, BLUEPRINT_ID } = options;
+
+const mockOptions = {
+  [PROJECT_ID]: 'project0',
+  [ENVIRONMENT_NAME]: 'environment0',
+  [ORGANIZATION_ID]: 'organization0',
+  [BLUEPRINT_ID]: 'blueprint0'
+};
 
 const mockGetEnvironment = jest.fn();
 const mockCreateEnvironment = jest.fn();
 const mockDeployEnvironment = jest.fn();
 
 jest.mock('../../src/lib/deploy-utils');
+jest.mock('../../src/lib/logger');
```

Comment:
why mock the logger?

Reply:
It bugged me to see its prints when running tests 😅